packages/patternfly-4/react-integration/demo-app-ts/src/Demos.ts

```diff
@@ -110,6 +180,16 @@ export const Demos: DemoInterface[] = [
     name: 'Label Demo',
     componentType: Examples.LabelDemo
   },
+  {
+    id: 'line-chart-color-demo',
```

Comment:
Noticed there are no tooltips here. Other than zoom, it would be ideal if we could ensure we have tooltips for all charts -- again, to show best practice.

Reply:
Sure! I pulled in your latest changes to the bar charts and line charts, which include tooltips.